types/src/grants.d.ts

```diff
@@ -18,6 +19,25 @@ export type GrantMetadata = {
   name: string;
   description: string;
   logoURI?: string;
+  quote?: string;
+  lastUpdated?: string;
```

Comment:
Per @apbendi in [Discord](https://discord.com/channels/562828676480237578/882302340655374437/882331205612240916), we should remove these two properties for now